docs/getting-started.rst

```diff
@@ -28,7 +28,7 @@ favorite tool on Linux) and run:
 
 All Systems::
 
-   mkdir path/to/my_game
+   mkdir -p path/to/my_game
```

Comment:
The `p` isn't strictly necessary, but adding it makes it cut-and-paste-able to the command line